client/src/App.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+
+// Components
+import User from "./components/User";
+import AddUser from "./components/AddUser";
+
+function App() {
+	// Users state
+	const [users, setUser] = useState([]);
+
+	// Get users on component did mount
+	useEffect(() => {
+		const fetchUsers = async () => {
```

Comment:
If you're going to do an async function you probably want to use a try/catch block inside the function so that you can catch the error if one is sent back from the server.